app/views/admin/agents/edit.html.slim

```diff
@@ -19,16 +19,21 @@
       .card-body.js_agent_role_form
         = simple_form_for [:admin, current_organisation, @agent] do |f|
           = render "model_errors", model: @agent
-          - allowed_to_change_services = Agent::TerritoryPolicy.new(current_agent, current_territory).allow_to_manage_access_rights?
 
-          = f.association :services, collection: @services, disabled: true, input_html: { class: "select2-input" }, wrapper_html: { class: "mb-0" }, hint: allowed_to_change_services ? nil : "Le changement de service est réservé aux admins de territoire"
-          - if allowed_to_change_services
-            span.small.text-muted
-              | Vous pouvez désormais attribuer plusieurs services à un agent, via votre interface de Configuration - Admin de territoire.&nbsp;
-              = link_to "Accéder à la configuration des agents", edit_admin_territory_agent_path(territory_id: current_territory.id, agent_id: @agent.id)
+          p
+            - if @agent.services.count == 1
+              | #{@agent.full_name} appartient à un seul service :
```

Comment:
c'est un joli détail d'avoir fait ce `if` 😍 . Tant qu'à faire, pour rendre ce cas un petit peu plus joli on pourrait même inliner le nom du service : 
```suggestion
              | #{@agent.full_name} appartient au service <b>#{@agent.services.first.name}</b>
```

Reply:
merci pour cette suggestion ! j’y avais réfléchi mais je m’étais dit que cette formulation et cette UI avec la liste indiquait mieux la possibilité d’appartenir à plusieurs services. je vais déployer tel quel et on pourrait itérer si ça te va ?